src/bin/pg_autoctl/ipaddr.c

```diff
@@ -244,3 +302,226 @@ isTestEnv(void)
 	const char *socketDir = getenv("PG_REGRESS_SOCK_DIR");
 	return socketDir != NULL && socketDir[0] == '\0';
 }
+
+
+/*
+ * From /Users/dim/dev/PostgreSQL/postgresql/src/backend/libpq/hba.c
+ */
+static bool
+ipv4eq(struct sockaddr_in *a, struct sockaddr_in *b)
+{
+	return (a->sin_addr.s_addr == b->sin_addr.s_addr);
+}
+
+
+/*
+ * From /Users/dim/dev/PostgreSQL/postgresql/src/backend/libpq/hba.c
+ */
+static bool
+ipv6eq(struct sockaddr_in6 *a, struct sockaddr_in6 *b)
+{
+	int			i;
+
+	for (i = 0; i < 16; i++)
```

Comment:
missing braces

Reply:
The bracing policy in PostgreSQL is different from ours... added now!